### PR TITLE
Add the beginnings of the readme and the code (solution not working).…

### PR DESCRIPTION
**Pull request by nathanrhead**
… Then internet died.

### CODE REVIEW

**Comment by nathanrhead** [comment]
- [] Top-level README “Table of Contents” is updated
 - [] Feature tasks for this challenge are completed
 - [] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: javascript/blogs/quick-sort/quick-sort.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Conclusion:
+
+## Efficiency
+
+- Time = O(n^2): The comparison happens n * (n-1) number of times, making the algorithm n-squared for time.
+- Space: O(1): The array is sorted in place, so no new array is created, making the space used constant.
+
+## References
+
+
```
Comment (adrianhuebner) [inline javascript/blogs/quick-sort/quick-sort.md:68]: You're off to a great start, completely understandable with the internet/power outage issues, come back to this when you have time!
